dashboard/dashboard-front/src/components/jobs/deployment/DeployForm.vue

```diff
@@ -0,0 +1,96 @@
+<script setup lang="ts">
+import { ref, type Ref } from 'vue'
+import { useRouter } from 'vue-router'
+import yaml from 'js-yaml'
+import { apiClient } from '@/services/ApiClient'
+import { toastService } from '@/services/ToastService'
+import { progressService } from "@/services/ProgressService"
+
+const yamlManifestRef: Ref<string> = ref('')
+const gitUsername = ref('')
+const gitPassword = ref('')
+const forceEnabled: Ref<boolean> = ref(false)
```

Comment:
Won't it automatically assume you want to use string in line 9 and boolean in line 12? Ref<> shouldn't be required.

Reply:
Right, it makes sense to keep it consistent